Mathlib/GroupTheory/Divisible.lean

```diff
@@ -161,6 +162,16 @@ instance Prod.rootableBy : RootableBy (B × B') β where
 
 end Prod
 
+section ULift
+
+@[to_additive]
+instance ULift.rootableBy [RootableBy A α] : RootableBy (ULift A) α where
+  root := λ x a ↦ ULift.up <| RootableBy.root x.down a
+  root_zero := λ x ↦ ULift.ext _ _ <| RootableBy.root_zero x.down
+  root_cancel := λ _ h ↦ ULift.ext _ _ <| RootableBy.root_cancel _ h
```

Comment:
The style guide says not to use `λ`, can you use `fun` instead?

Reply:
(throughout)

Reply:
done (I think)